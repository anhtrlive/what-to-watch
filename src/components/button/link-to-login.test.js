import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import LinkLoginPlusList from './link-to-login';
import {BrowserRouter} from 'react-router-dom';

Enzyme.configure({adapter: new Adapter()});


it(`link to login`, () => {
  const tree = renderer.create(
      <BrowserRouter>
        <LinkLoginPlusList />
      </BrowserRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
