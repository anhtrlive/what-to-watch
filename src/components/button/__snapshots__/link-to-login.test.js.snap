// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link to login 1`] = `
<a
  className="btn btn--list movie-card__button"
  href="/login"
  onClick={[Function]}
>
  <svg
    height={20}
    viewBox="0 0 19 20"
    width={19}
  >
    <use
      xlinkHref="#add"
    />
  </svg>
  <span>
    My list
  </span>
</a>
`;
